api/models/User.js

```diff
@@ -221,7 +229,31 @@ module.exports = bookshelf.Model.extend(merge({
     DELETE FROM posts_tags WHERE post_id in (select id from posts WHERE user_id = ${this.id});
     DELETE FROM groups_posts WHERE post_id in (select id from posts WHERE user_id = ${this.id});
     DELETE FROM votes WHERE user_id = ${this.id};
-    DELETE FROM users WHERE id = ${this.id};
+
+    UPDATE users SET 
+    active = false, 
```

Comment:
TBD

Reply:
What are our options here if we flip back to active, considering we would want to turn off notifications? When sending notifications, check the user name and if it matches "Deleted User", filter it out? Are there other things we'd want to skip doing?

Alternatively, if its just that we want posts and comments from deleted users to still show up, do we just change the current queries for those to now filter out non-active users?? What non-active users do we actually not want to see posts or comments for?

Reply:
yes it seems like the whole point is to have the posts and comments still show up. but im not sure we want them to show up for people who mark themselves as deactivated. again, lets talk it through